zebra/zserv.h

```diff
@@ -185,6 +185,9 @@ struct zserv {
 	uint32_t local_es_evi_add_cnt;
 	uint32_t local_es_evi_del_cnt;
 	uint32_t error_cnt;
+	uint32_t l3nhg_add_cnt;
```

Comment:
Can it be L2 nhg at all? Asking because what role does the L3 prefix play here.

Reply:
It can be but I believe we are quibling a bit.  I wouldn't worry about this at all.

Reply:
> Can it be L2 nhg at all? Asking because what role does the L3 prefix play here.

mmm . actually a nexthop is a nexthop group.
I just try to stick with what the system offers.

unless you explain me what is a L2 nexthop..

Reply:
>It can be but I believe we are quibling a bit. I wouldn't worry about this at all.

Not sure I understood the comment correctly :) Does it mean we should keep `l3nhg_*` or use `nhg_*`?

Reply:
updated with nhg_* as everyone can understand it as desired